artiq/coredevice/core.py

```diff
@@ -153,6 +153,17 @@ def mu_to_seconds(self, mu):
     def get_rtio_counter_mu(self):
         return rtio_get_counter()
 
+    @kernel
+    def wait_until_rtio_counter_mu(self, cursor_mu):
```

Comment:
Bikeshed: Maybe `wait_until_mu(timestamp_mu)` and maybe add `wait_until(timestamp)` in seconds.

Reply:
Agreed

Reply:
Except we should not have ``wait_until``- all absolute timestamps in ARTIQ are in mu.